### PR TITLE
AdminManageComments#011

### PR DESCRIPTION
**Pull request by Salim-54**
#### What does this PR do?

- Admin comments manage page

#### Description of Task to be completed? 

As a user, I should be able to see
- All comments 
- the date they have been sent
- their senders
- the delete button
- reply button 

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``AdminManageComments#011``

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker/trello stories?

- [link to my pivotal tracker/tello/jira ... implemented story](https://trello.com/c/nb0S66k5/29-adminmanagecomments011)

#### Screenshots (if appropriate)
- 
Screenshot (269)

#### Questions:

- N/A

### CODE REVIEW

File: css/article.css [hunk 1]
```diff
@@ -284,3 +284,234 @@ body {
...
+    position: fixed;
+    top: 0;
+    left: 0;
+    opacity: 0;
+    visibility: hidden;
+    transition: all .3s; }
+  
+  /* .popup:target {
+    opacity: 1;
+    visibility: visible; } */
```
Comment (thepatrickniyo) [inline css/article.css:312]: Remove comments

File: css/articles.css [hunk 2]
```diff
@@ -204,3 +204,232 @@ body {
...
+
+  /* CODES FOR POPUP */
+
+
+  .popup {
+    width: 100vw;
+    height: 100vh;
+  
+    /* width: 100%; */
+    /* height: 100vh; */
```
Comment (thepatrickniyo) [inline css/articles.css:218]: Comments as well

File: css/editnew.css [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+}
+
+/* .editor__inputs-btn{
+  padding: 1rem 2rem;
+  border: none;
+  background-color: #525D62;
+  color: #EEF5F7;
+  border-radius: 80rem;
+} */
```
Comment (thepatrickniyo) [inline css/editnew.css:234]: Comments as well

File: css/style.css [hunk 4]
```diff
@@ -513,39 +513,30 @@ p.about__text {
...
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  z-index: 10;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-items: center; }
+  visibility: visible; } */
+
```
Comment (PatrickNiyogitare28) [inline css/style.css:539]: comments

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
The implementation is great, but please take care where you're leaving commented codes behind. Remove them to increase the code quality

**Pull request merged**